xtask/codegen/css.ungram

```diff
@@ -1719,6 +1730,9 @@ CssRatio =
 AnyCssFunction =
 	CssFunction
 	| CssUrlFunction
+	| CssTailwindAlphaFunction
+	| CssTailwindSpacingFunction
+	| CssTailwindValueFunction
```

Comment:
Very much like the other languages (JS and HTML), they could be called `Tailwind*`